drivers/filesystems/fastfat_new/fatstruc.h

```diff
@@ -1370,8 +1370,12 @@ typedef struct _CCB {
     //  Define a 24bit wide field for Flags, but a UCHAR for Wild Cards Present
     //  since it is used so often.  Line these up on byte boundaries for grins.
     //
-
```

Comment:
Why do you remove this blank line?

Reply:
In my opinion its good move, because comment line above is about variable that had be changed, so it clearly mean that this block of code is connected to comment higher